docs/pages/operations-guide.md

```diff
@@ -851,9 +851,54 @@ $ curl -k -X POST -H "Authorization: token=$(dcos config show core.dcos_acs_toke
 }
 ```
 
+## Stop a pod
+
+Stopping a pod relaunches it in an idle command state. This allows the operator to debug the contents of the pod, possibly making changes to fix problems. While these problems are often fixed by just replacing the pod, there may be cases where an in-place repair or other operation is needed. For example:
+- A pod which crash-loops as soon as it's started may need additional work to be performed.
+- Some services may _require_ that certain repair operations be performed manually when the task itself isn't running.
+Being able to put the pod in an offline but accessible state makes it easier to resolve these situations.
+
+After the pod has been stopped, it may be started again, at which point it will be restarted again and will resume running task(s) where it left off.
+
+Here is an example session where an `index-1` node is crash looping due to some corrupted data in a persistent volume. The operator stops the `index-1` pod, then uses `task exec` to repair the index. Following this, the operator starts the pod and it resumes normal operation:
```

Comment:
The example session below makes it look like commands take effect instantaneously.  Maybe put some `STOPPING` and `STARTING` status messages in there.